workflow_tests/assertions/verifySignedCommitsAssertions.ts

```diff
@@ -0,0 +1,17 @@
+import type {Step} from '@kie/act-js';
+import {createStepAssertion} from '../utils/utils';
+
+function assertVerifySignedCommitsJobExecuted(workflowResult: Step[], didExecute = true) {
+    const steps = [createStepAssertion('Verify signed commits', true, null, 'VERIFYSIGNEDCOMMITS', 'Verify signed commits', [{key: 'GITHUB_TOKEN', value: '***'}], [])] as const;
+
+    steps.forEach((expectedStep) => {
+        if (didExecute) {
+            expect(workflowResult).toEqual(expect.arrayContaining([expectedStep]));
+        } else {
+            expect(workflowResult).not.toEqual(expect.arrayContaining([expectedStep]));
+        }
+    });
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export {assertVerifySignedCommitsJobExecuted};
```

Comment:
some of the files in this directory are pre-generated, so to keep the interface uniform I'd rather keep it this way instead of differentiating between `export default and export {...}`